frontend/src/components/pipelines-or-deployments/etl-task-deploy/EtlTaskDeploy.jsx

```diff
@@ -95,21 +95,32 @@ const EtlTaskDeploy = ({
       return null;
     });
   };
-
-  const getWorkflows = () => {
-    const connectorType = type === "task" ? "FILESYSTEM" : "DATABASE";
+  const fetchWorkflows = (type) =>
     workflowApiService
-      .getWorkflowEndpointList("DESTINATION", connectorType)
-      .then((res) => {
-        const updatedData = res?.data.map((record) => ({
+      .getWorkflowEndpointList("DESTINATION", type)
+      .then((res) =>
+        res?.data.map((record) => ({
           ...record,
           id: record.workflow,
-        }));
-        setWorkflowList(updatedData);
-      })
+        }))
+      )
       .catch(() => {
-        console.error("Unable to get workflow list");
+        console.error(`Unable to get workflow list for ${type}`);
```

Comment:
@vishnuszipstack Do we need this `console.error`? Instead, can we add the [alert pop message](https://github.com/Zipstack/unstract/blob/4c97569e03347d6bf75f0aa75d5ec59aa5b5d7d5/frontend/src/components/custom-tools/add-llm-profile/AddLlmProfile.jsx#L237)?

Reply:
not required. removed